doc/languages-frameworks/dotnet.section.md

```diff
@@ -133,7 +133,7 @@ For more detail about managing the `deps.nix` file, see [Generating and updating
 * `dotnetPackFlags` can be used to pass flags to `dotnet pack`. Used only if `packNupkg` is set to `true`.
 * `dotnetFlags` can be used to pass flags to all of the above phases.
 
-When packaging a new application, you need to fetch its dependencies. Create an empty `deps.nix`, set `nugetDeps = ./deps.nix`, then run `nix-build -A package.fetch-deps` to generate a script that will build the lockfile for you.
+When packaging a new application, you need to fetch its dependencies. Create an empty `deps.json`, set `nugetDeps = ./deps.json`, then run `nix-build -A package.fetch-deps` to generate a script that will build the lockfile for you.
```

Comment:
Not that related to this PR, but when I started with dotnet in nix I found it confusing how to properly generate/update the deps file (I think the documentation didn't even have this line at that time)
```suggestion
When packaging a new application, you need to fetch its dependencies. Create an empty `deps.json`, set `nugetDeps = ./deps.json`, then run `$(nix-build -A package.fetch-deps)` to generate the lockfile.
```

Reply:
> when I started with dotnet in nix I found it confusing how to properly generate/update the deps file (I think the documentation didn't even have this line at that time)

That's good feedback. :heart: 

> then run `$(nix-build -A package.fetch-deps)` to generate the lockfile.

This is very bash/posix-centric, which may not be a problem, but perhaps it'd be better to say something like "build `fetch-deps` and run the result"?

I would propose breaking this done more clearly, something like:

```markdown
When packaging a new application, you must fetch its dependencies.
You can do this as follows:

1. Create an empty `deps.json` alongside your `package.nix`
2. Add `nugetDeps = ./deps.json` to your `package.nix`
3. Build the fetch script: `nix-build -A your-package.fetch-deps`
4. Run the script: `./result`

::: {.note}
`buildDotnetModule` adds the `fetch-deps` package as a "passthru" attribute on your package.
:::
```

---

However this could be in a follow up PR, as I don't see why it should block this one.